.github/workflows/ci.yml

```diff
@@ -175,3 +264,44 @@ jobs:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
           SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
           FLAKY_UNIT_TESTS: ${{needs.check-results.outputs.flakyUnitTests}}
+
+
+  deploy-camunda-docker-snapshot:
+    name: Deploy snapshot Camunda Docker image
+    needs: [ detect-changes, check-results ]
+    runs-on: ubuntu-latest
```

Comment:
❓ 🔧 Could we reuse the image from before and just retag it?

Reply:
I am afraid we can't, this is a diffrent Job and can be run by a different worker
the image built in `docker-checks` job is pushed to a local registery, that must be killed after the job is completed

Reply:
We could maybe via caching or using job artifacts I think: https://github.com/camunda/github-actions-recipes#share-data-between-jobs-via-actionsupload-artifact-and-actionsdownload-artifact-usages-in-camunda

Reply:
@cmur2 @Zelldon the same can be done in [zeebe-ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/zeebe-ci.yml)
I created this issue https://github.com/camunda/camunda/issues/19865 for the improvement.